api/v1/openapi.yaml

```diff
@@ -224,6 +224,27 @@ paths:
           x-go-name: Failed
           schema:
             "$ref": "#/definitions/Error"
+  "/endpoints/":
```

Comment:
`/endpoint/` returns one endpoint, while `/endpoints/` returns an array of endpoints. I'm open to moving this under `/endpoint/`, though and changing `/endpoint/` to return a list of endpoints as well.